From: Corey Wright <undefined@pobox.com>
Date: Tue, 6 Mar 2012 11:10:30 +0900
Subject: Adds support for DVD-ROM medium-type.

Description: Adds support for DVD-ROM medium-type.
Author: Corey Wright <undefined@pobox.com>
Last-Update: 2012-03-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 scsi-layer.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/scsi-layer.c b/scsi-layer.c
index 6717b12..64a3dba 100644
--- a/scsi-layer.c
+++ b/scsi-layer.c
@@ -908,6 +908,11 @@ static int query_dvd(DeviceHandle *dh, int probe_only)
 	    break;
 	 }
 
+	 if(layer_type & 0x01)
+	 {  dh->typeDescr = g_strdup("DVD-ROM");
+	    break;
+	 }
+
 	 if(layer_type & 0x06) /* strange thing: (re-)writeable but neither plus nor dash */ 
 	 {  dh->typeDescr = g_strdup("DVD-ROM (fake)");
 	    dh->subType = DVD;
